mirrord/cli/src/main.rs

```diff
@@ -162,15 +149,7 @@ async fn exec(args: &ExecArgs, progress: &TaskProgress) -> Result<()> {
 
     let config = LayerConfig::from_env()?;
 
-    if config.pause {
-        if config.agent.ephemeral {
-            error!("Pausing is not yet supported together with an ephemeral agent container.");
-            panic!("Mutually exclusive arguments `--pause` and `--ephemeral-container` passed together.");
-        }
-        if !config.feature.network.incoming.is_steal() {
-            warn!("{PAUSE_WITHOUT_STEAL_WARNING}");
-        }
-    }
+    config.verify()?;
```

Comment:
What do you say about moving this to `MirrordExecution::start`, so that verification is also done when running from an IDE? It could be called [where the targetless config checks are done now](https://github.com/metalbear-co/mirrord/blob/79061d1e755c0f7da86e1fc1da9f222df8440b74/mirrord/cli/src/execution.rs#L60), and those checks could be moved into `verify()`.